pkg/apis/sources/v1alpha1/rabbitmq_types.go

```diff
@@ -124,6 +125,22 @@ type RabbitmqSourceSpec struct {
 	// RabbitMQ cluster
 	// +optional
 	Predeclared bool `json:"predeclared,omitempty"`
+	// Retry is the minimum number of retries the sender should attempt when
+	// sending an event before moving it to the dead letter sink.
+	// +optional
+	Retry *int32 `json:"retry,omitempty"`
```

Comment:
Would this and `BackoffPolicy`,  `BackoffDelay` be available for users to configure or they'd have fixed values in code but wouldn't be configurable by users ?

Reply:
After what we talked about today, I think this makes sense (they are configurable by users but have a default). Also the Eventing Source Spec "explicitly permit" additional data in the Source's Spec and Status (https://github.com/knative/eventing/blob/main/docs/spec/sources.md#source-custom-objects)